client/v3/config.go

```diff
@@ -90,3 +93,103 @@ type Config struct {
 
 	// TODO: support custom balancer picker
 }
+
+type ClientConfig struct {
+	Endpoints        []string      `json:"endpoints"`
+	RequestTimeOut   time.Duration `json:"request-timeout"`
+	DialTimeout      time.Duration `json:"dial-timeout"`
+	KeepAliveTime    time.Duration `json:"keepalive-time"`
+	KeepAliveTimeout time.Duration `json:"keepalive-timeout"`
+	Secure           *SecureConfig `json:"secure"`
+	Auth             *AuthConfig   `json:"auth"`
+}
+
+type SecureConfig struct {
+	Cert       string `json:"cert"`
+	Key        string `json:"key"`
+	Cacert     string `json:"cacert"`
+	ServerName string `json:"servername"`
```

Comment:
```suggestion
	ServerName string `json:"server-name"`
```

Reply:
Ah-:), I changed the name between "servername" and "server-name" several times back and forth. In the beginning, I used `server-name`, but I see "username", so I changed it back to "servername". 

Now you proposed "server-name" again, OK, then let me change it to "server-name".